### DISCUSSION TITLE
Divider dashes - color?

### DISCUSSION THREAD

**Discussion by georgebrianb** [saadeghi/daisyui#473] [Q&A] [answered]
Hey, kind of new to web dev, especially front-end, and I'm struggling to change the color of the lines before and after the text inside the divider. I'm talking about these:

![image](https://user-images.githubusercontent.com/32679117/154431322-0aa8afda-b57d-49c5-8219-d0d44890c6dc.png)

Looking forward for some help

**Comment by saadeghi** [comment, marked as answer]
@georgebrianb 
```html
<div class="divider before:bg-primary after:bg-secondary">
```
https://play.tailwindcss.com/jqP28O0jcE

**Reply by georgebrianb** [reply]
Sweet, and given that you're already here, what happened to the form examples in the docs? They were really useful, and there were a lot of them. If those are gone, some example login/signup screens would be amazing. Thanks for the help!

**Reply by saadeghi** [reply]
I just rearranged the sidebar items. They're under **Data input** section:  
https://daisyui.com/components/input/  
<img width="324" alt="Screen Shot 2022-02-17 at 11 38 09 AM" src="https://user-images.githubusercontent.com/7342023/154437459-14c3b6e3-d4e3-45b0-9ec4-c4e1c2721c62.png">

**Reply by georgebrianb** [reply]
Hmm, I still can't find the form examples that were designed using a card / inputs / buttons etc. But thanks for pointing it out for me!

**Reply by AshSimmonds** [reply]
Well the cached version is still there (edit: not fully functional though):

https://webcache.googleusercontent.com/search?q=cache:M3qKCq2ohnAJ:https://daisyui.com/components/form/input/+&cd=1&hl=en&ct=clnk&gl=au

**Reply by saadeghi** [reply]
@georgebrianb Here's an example:  
https://play.tailwindcss.com/XpssSsGRf2
